.github/workflows/format.yml

```diff
@@ -0,0 +1,39 @@
+name: Format Code
+
+on:
+  pull_request:
+    branches: 
+      - '*'
+
+jobs:
+  format:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        ref: ${{ github.head_ref }}
+        fetch-depth: 0
+    
+    - name: Use Node.js
+      uses: actions/setup-node@v3
+      with:
+        node-version: '18'
+    
+    - name: Install dependencies
+      run: npm ci --legacy-peer-deps
```

Comment:
Can you explain why you need this flag? I see that its usage is not recommended: https://docs.npmjs.com/cli/v10/using-npm/config#legacy-peer-deps

Reply:
Good point. Thank you for raising it.  `@algolia/ui-components-horizontal-slider-react@1.2.2` was not compatible with React 18, and I reported it to the front-end team. The front-end team said it is safe to use `--legacy-peer-deps` for now, but it also seems they released 1.2.3, and it is working fine without the flag.